src/algokit.ts

```diff
@@ -0,0 +1,294 @@
+import algosdk from 'algosdk'
+import { assetOptIn, assetOptOut } from './asset'
+import {
+  getAlgoClient,
+  getAlgoIndexerClient,
+  getAlgoKmdClient,
+  getAlgoNodeConfig,
+  getConfigFromEnvOrDefaults,
+  getDefaultLocalNetConfig,
+} from './network-client'
+import { getTransactionParams, performAtomicTransactionComposerSimulate, sendAtomicTransactionComposer } from './transaction'
+import { transferAlgos } from './transfer'
+import { AlgoAmount } from './types/amount'
+import { AlgoConfig } from './types/network-client'
+import { TransactionNote } from './types/transaction'
+import Algodv2 = algosdk.Algodv2
+import Indexer = algosdk.Indexer
+import Kmd = algosdk.Kmd
+import AtomicTransactionComposer = algosdk.AtomicTransactionComposer
+import SuggestedParams = algosdk.SuggestedParams
+import Transaction = algosdk.Transaction
+import TransactionSigner = algosdk.TransactionSigner
+import modelsv2 = algosdk.modelsv2
+
+export interface AlgoClients {
+  algod: Algodv2
+  indexer?: Indexer
+  kmd?: Kmd
+}
+
+export type AccountAddress = string
+
+export type TransactionSent = {
+  /** The transaction */
+  transaction: Transaction
+  /** The response if the transaction was sent and waited for */
+  confirmation: modelsv2.PendingTransactionResponse
+}
+
+export class AlgoKit {
```

Comment:
This isn't what the final code would look like, it's a quick and dirty single file prototype